blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,344 @@
+  const url = websites[arcSite].website_url;
+  return (
+    <div className="list-item" ref={ref}>
+      {(promoElements.showImage)
+        ? (
+          <div className="results-list--image-container mobile-order-2 mobile-image">
+            <a
+              href={url}
+              title={headlineText}
+              aria-hidden="true"
+              tabIndex="-1"
+            >
+              <Image
+                {...imageProps}
+                url={imageURL !== null ? imageURL : targetFallbackImage}
+                alt={imageURL !== null ? headlineText : imageProps.primaryLogoAlt}
+                resizedImageOptions={imageURL !== null
+                  ? extractResizedParams(element)
+                  : placeholderResizedImageOptions}
+              />
+            </a>
+          </div>
+        )
+        : null}
+      {(promoElements.showHeadline)
+        ? (
+          <div className="results-list--headline-container mobile-order-1">
+            <a href={url} title={headlineText}>
+              <Heading className="headline-text">{headlineText}</Heading>
+            </a>
+          </div>
+        )
+        : null }
+      {(promoElements.showDescription || promoElements.showDate || promoElements.showByline)
+        ? (
+          <div className="results-list--description-author-container mobile-order-3">
+            {(promoElements.showDescription && descriptionText)
+              ? (
+                <a href={url} title={headlineText}>
+                  <SecondaryFont as="p" className="description-text">
+                    {descriptionText}
+                  </SecondaryFont>
+                </a>
+              )
+              : null}
+            {(promoElements.showDate || promoElements.showByline)
+              ? (
+                <div className="results-list--author-date">
+                  {(promoElements.showByline)
+                    ? (
+                      <Byline
+                        content={element}
+                        list
+                        separator={promoElements.showDate}
+                        font="Primary"
+                      />
+                    )
+                    : null }
+                  {(promoElements.showDate)
+                    ? <ArticleDate classNames="story-date" date={displayDate} />
+                    : null }
+                </div>
+              )
+              : null}
+          </div>
+        )
+        : null }
+    </div>
+  );
+}));
+
+const Results = () => {
+  const {
+    arcSite,
+    contextPath,
+    customFields,
+    deployment,
+    isAdmin,
+  } = useFusionContext();
+
+  const {
+    lazyLoad,
+    listContentConfig: {
+      contentService,
+      contentConfigValues,
+    },
+  } = customFields;
+
+  const [{
+    breakpoints,
+    fallbackImage,
+    locale,
+    primaryLogoAlt,
+    resizerURL,
+  }] = useState(getProperties(arcSite) || {});
+
+  const [imageProps] = useState({
```

Comment:
Lifting this to the parent and passing as prop would remove the need for `useState`

Reply:
Makes sense with the other changes.